### PR TITLE
GitHub Research Docs Fixes

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Fresh PR for Camilla's work fixing a number of broken links in the research syntheses folder, namely:
- Updating older cross reference links between synthesis documents to adopt more recent changes in naming convention that broke links
- Removing a couple links to files that don't exist in the repo anymore
- Updating a few wiki links to point toward repo documents instead 
- Fixing jump-to heading link syntax within syntheses

Relevant folder of branch can be browsed here https://github.com/raft-tech/TANF-app/tree/github-ux-sytheses-fixes/docs/User-Experience/Research-Syntheses

### How to test
All links highlighted within files changed should now be functional

### CODE REVIEW

File: docs/User-Experience/Research-Syntheses/2020, Fall - Flatfile Metadata Guide.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ For tribes, ### represents their 3 digit Tribe Code
 
  
 
-🔍 **Research**: There are open questions and we’d like to base a recommendation on insights from Phase III Research ([October STT Research](https://github.com/HHS/TANF-app/blob/main/docs/User-Research/2020%2C%20Fall%20-%20Understanding%20STT%20Roles%2C%20Source%20of%20Truth%2C%20and%20Metadata.md)) and continued solutions ideation (Both design and technical).
+🔍 **Research**: There are open questions and we’d like to base a recommendation on insights from Phase III Research ([October STT Research](https://github.com/HHS/TANF-app/blob/main/docs/User-Experience/Research-Syntheses/2020%2C%20Fall%20-%20Understanding%20STT%20Roles%2C%20Source%20of%20Truth%2C%20and%20Metadata.md) and continued solutions ideation (Both design and technical).
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Fall - Flatfile Metadata Guide.md]:
```suggestion
🔍 **Research**: There are open questions and we’d like to base a recommendation on insights from Phase III Research ([October STT Research](https://github.com/HHS/TANF-app/blob/main/docs/User-Experience/Research-Syntheses/2020%2C%20Fall%20-%20Understanding%20STT%20Roles%2C%20Source%20of%20Truth%2C%20and%20Metadata.md)) and continued solutions ideation (Both design and technical).
```

File: docs/User-Experience/Research-Syntheses/2021, Spring - Evil User Journeys & TANF Data Errors.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ Our Error Data workshops also identified linking element errors (which are relat
 **Project Impact(s)**
 
 - We prioritized Dan and Malcolm for use in short term follow-on workshops; these could potentially include stakeholders from outside the project team so as to better identify risks and collaborate on controls.  
-- The [evil user journey maps]() :lock: synthesized from both workshops can be used to generate acceptance criteria for continued work on TDP. 
+- The [evil user journey maps](https://github.com/HHS/TANF-app/blob/main/docs/User-Experience/Research-Syntheses/2021%2C%20Spring%20-%20Evil%20User%20Journeys%20%26%20TANF%20Data%20Errors.md) :lock: synthesized from both workshops can be used to generate acceptance criteria for continued work on TDP.
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2021, Spring - Evil User Journeys & TANF Data Errors.md]:
```suggestion
- The [evil user journey maps](https://github.com/HHS/TANF-app/blob/main/docs/User-Experience/Research-Syntheses/2021%2C%20Spring%20-%20Evil%20User%20Journeys%20%26%20TANF%20Data%20Errors.md) synthesized from both workshops can be used to generate acceptance criteria for continued work on TDP. 
```

File: docs/User-Experience/Research-Syntheses/2022, Fall - TDP 2.2 Pilot Expansion.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ Based on the task scenarios defined for v2.2 the pilot expansion program was a s
 
 ### Retest observed gaps from August 2022 pilot program 
 
-In addition to reviewing new functionality, we wanted to revisit and retest the areas noted from the August 2022 pilot program. Please reference [August 2022](https://tdp-project-updates.app.cloud.gov/august-2022-update.html) for full details 
+In addition to reviewing new functionality, we wanted to revisit and retest the areas noted from the August 2022 pilot program. Please reference [August 2022](https://github.com/HHS/TANF-app/blob/main/docs/User-Experience/Research-Syntheses/2022%2C%20Fall%20-%20TDP%202.2%20Pilot%20Expansion.md) for full details
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2022, Fall - TDP 2.2 Pilot Expansion.md]:
```suggestion
In addition to reviewing new functionality, we wanted to revisit and retest the areas noted from the August 2022 pilot program. Please reference [August 2022](https://github.com/HHS/TANF-app/blob/main/docs/User-Experience/Research-Syntheses/2022%2C%20Summer%20-%20TDP%202.0%20Pilot%20Program.md) for full details 
```
I think we want to link to August 2022 instead but please double check

File: docs/User-Experience/Research-Syntheses/2023, Winter - TDP 3.0 Pilot Program.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 #2023, Winter - TDP 3.0 Pilot Program
 ===
 
-Following the [January 2023 research session](https://github.com/raft-tech/TANF-app/blob/develop/docs/User-Experience/Research-Syntheses/2022%2C%20Winter%20-%20Understanding%20How%20STTs%20Use%20the%20Transmission%20Report.md) on error message handling and resolution, the team transitioned into supporting the February 2023 pilot expansion cohort. We learned the following key insights from participating states, tribes, and territories (STT) and TDP system admin:
+Following the [January 2023 research session](https://github.com/HHS/TANF-app/blob/main/docs/User-Experience/Research-Syntheses/2023%2C%20Winter%20-%20TDP%203.0%20Pilot%20Program.md) on error message handling and resolution, the team transitioned into supporting the February 2023 pilot expansion cohort. We learned the following key insights from participating states, tribes, and territories (STT) and TDP system admin:
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2023, Winter - TDP 3.0 Pilot Program.md]: Is this right? It's self referential. Should it be https://github.com/HHS/TANF-app/blob/main/docs/User-Experience/Research-Syntheses/2022%2C%20Winter%20-%20Understanding%20How%20STTs%20Would%20Use%20the%20Transmission%20Report%20copy.md

**Review by lfrohlich — CHANGES_REQUESTED** [review]
Made a few comments, otherwise looks good

**Comment by reitermb** [comment]
@lfrohlich All those fixes should be addressed now + one more formatting fix I found!

**Review by lfrohlich — APPROVED** [review]
Thanks for making these changes.

**Pull request merged**